src/xg.cpp

```diff
@@ -258,6 +272,163 @@ void XG::load(istream& in) {
     }
 
 }
+    
+void XG::convert_old_edge_to_new(int_vector<> g_iv_old, bit_vector g_bv_old, rank_support_v<1> g_bv_rank_old, bit_vector::select_1_type g_bv_select_old) {
```

Comment:
These arguments should be passed in by reference to avoid deep copying the vectors